docs/changelog.md

```diff
@@ -1,5 +1,19 @@
 ## Changelog
 
+### 3.4.0
+
+* SECURITY: WordPress can create images for PDF documents which if used would leak the hidden document name so image name changed.
```

Comment:
Are there any remediation steps necessary for users who previously created an image for a PDF document?

Reply:
If the image is to be used within a shortcode/block, then before doing so, it will check if the name of the image has been changed. If it has, then it can be used. If not, then the image names are changed and then it is displayed.
(Routine hide_doc_attach_slug used to change them on loading; hide_exist_doc_attach_slug used later.)
The REST interface is more simplistic since it is not going to display the image. If it has not been changed, then the image names are hidden.